### ISSUE TITLE
Details about training and prediction on the BEAT dataset

### ISSUE THREAD

**Issue by YiJin-Chen** [uuembodiedsocialai/FaceDiffuser#7]
Could you please provide some specific code for training and prediction on the BEAT dataset? It seems that this part is not included in the current code.

I tried to modify the code you provided to reproduce the results on the BEAT data set, but the test results were far from those reported in the paper.

It would be appreciated that if you could provide the relevant code and model training weights for the BEAT dataset.

![e7f1bee0c80cb2c50fc8496436589dd](https://github.com/uuembodiedsocialai/FaceDiffuser/assets/128764400/fb790bc4-719a-4fea-96df-01b8c6b8edd0)

**Comment by YiJin-Chen** [comment]
OK, thanks

**Comment by hslr4** [comment]
I have used the provided instructions to train on BEAT but still cannot reproduce the results from the paper:

```
Frame Number: 75600
Mean Vertex Error: 8.6811e-01
Lip Vertex Error: 3.0576e-01
FDD: 1.2823e-01
ABS FDD: 1.5363e-01
```

However the validation loss decreases during training so training seems to work.

![Image](https://github.com/user-attachments/assets/3aba8b70-582d-4eb3-afc1-b5882fa1adf1)

Thus I think there is a problem with prediction (`test_diff`). 
Did you face similar problems @YiJin-Chen ? Any ideas what is going wrong @stefan-st ?
Any help would be much appreciated :)
